editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2139,12 +2134,7 @@ void Node3DEditorViewport::_nav_look(Ref<InputEventWithModifiers> p_event, const
 		cursor.x_rot += p_relative.y * radians_per_pixel;
 	}
 	cursor.y_rot += p_relative.x * radians_per_pixel;
-	if (cursor.x_rot > Math_PI / 2.0) {
-		cursor.x_rot = Math_PI / 2.0;
-	}
-	if (cursor.x_rot < -Math_PI / 2.0) {
-		cursor.x_rot = -Math_PI / 2.0;
-	}
+	cursor.x_rot = CLAMP(cursor.x_rot, -Math_PI / 2, Math_PI / 2.0005);
```

Comment:
```suggestion
	cursor.x_rot = CLAMP(cursor.x_rot, -1.57, 1.57);
```

I think we can just hard-code the values instead of changing the number we divide by.